### ISSUE TITLE
Make an user guide

### ISSUE THREAD

**Issue by Andre9642** [Andre9642/BrailleExtender#43]
### The problem
The current documentation is not a documentation but a listing of braille gestures mapped by the add-on profiles. Also, the add-on contains several features that should be documented.

### Suggested solution
In the first time, I'll write a French documentation (from the 21st).
Then I'll ask for help with the English translation (due to my English not being good). I hope to integrate the English and French documentation in mid-January.
The current documentation will be renamed something like “Profile gestures list”.

**Comment by zstanecic** [comment]
Hi,

I am tempted to close this, as the current documentation describes the usage of the add-on well.

Even Russian, polish and Croatian documentation are good translated examples.

It’s not because I have did so, but it’s because I had feedback from users.

**Comment by Andre9642** [comment]
I disagree. In my opinion, the current documentation is too minimalist.
Each feature should be detailed (including a description, use cases and/or examples).
Features such as braille dictionaries or HUC Braille may not be obvious for the user without some basic explanation.

**Comment by zstanecic** [comment]
Yes, but this should be translated in a .po file like before, with the interface.

Along with this, please include the comments for translators

**Comment by Andre9642** [comment]
My final goal is to include the add-on to the translation system.
So, I'll write the documentation in Markdown and I'll add comments for translators, yes.

**Comment by zstanecic** [comment]
But try to add the existing strings when adding it to the translation system, as I don’t want my work to be destroyed, cause there are glossary terms and consictent translations

**Comment by Andre9642** [comment]
Don't worry about that. :) Most of strings should not change! :)
Thanks a lot (again) for your contributions.

**Comment by zstanecic** [comment]
I will continue with this…

**Comment by bdorer** [comment]
This would help a lot. For example smart Caps Lock isn't well described without a guide.
